internal/component/loki/process/stages/regex.go

```diff
@@ -112,7 +112,9 @@ func (r *regexStage) Process(labels model.LabelSet, extracted map[string]interfa
 			extracted[name] = match[i]
 		}
 	}
-	level.Debug(r.logger).Log("msg", "extracted data debug in regex stage", "extracted data", fmt.Sprintf("%v", extracted))
+    if level.DebugValue() {
```

Comment:
This function doesn't exist. Did you refactor it away and not commit the fully refactor?